backstory/src/main/webapp/visionUploadPrototype/visionDemo.js

```diff
@@ -0,0 +1,75 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/* exported getAnalyzedImages fetchBlobstoreUrl */
+
+
```

Comment:
As mentioned in the [JS Style Guide](https://google.github.io/styleguide/jsguide.html#file-name), file names should be all lowercase, but they can use underscores or dashes. Maybe ``vision-demo.js`` would be a better file name.

Reply:
Okay I renamed the folder, html, css, and javascript files to follow the style guide.

Reply:
Sounds great.